hikari/api/rest.py

```diff
@@ -338,7 +338,7 @@ async def edit_permission_overwrites(
         target : typing.Union[hikari.users.PartialUser, hikari.guilds.PartialRole, hikari.channels.PermissionOverwrite, hikari.snowflakes.Snowflakeish]
             The channel overwrite to edit. This may be the object or the ID of an
             existing overwrite.
-        target_type : hikari.undefined.UndefinedOr[typing.Union[hikari.channels.PermissionOverwriteType, builtins.str]]
```

Comment:
Should these have not been left?

Reply:
The consistency across all REST endpoint is force the args that take an enum dont also take the raw value. If you want to change this it might be worth to make another pr for it, since this one is getting kinda big.

TLDR; consistency

Reply:
Oh thought that was a model not a method (was looking on mobile)

Reply:
All good :)